backEnd/validations/userValidator.js

```diff
@@ -0,0 +1,42 @@
+const FluentValidation = require('@kalstong/fluentvalidation');
```

Comment:
Should validation be done on the client or server?

Reply:
both